### PR TITLE
Controller/electrical meter

### PR DESCRIPTION
**Pull request by aleemont1**
This PR refines the ElectricalMeter, in order to make it work in real time.

### CODE REVIEW

File: src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java [hunk 1]
```diff
@@ -86,4 +91,23 @@ public double getGlobalConsumption() {
...
+    }
+
+    @Override
+    public void setAveragePower(final double averagePower) {
+        this.averagePower = averagePower;
+    }
+
+    @Override
+    public void updateTick(final Duration deltaTime) {
+        final double deltaHours = deltaTime.toSeconds() / 3600;
```
Comment (michelececcacci) [inline src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java]: Please use `DurationConverter` (and also modify other instances of this in the code you previously wrote, after the PR). Minor, but brings consistency to the codebase. Everything else looks fine, so merge after you fix this

File: src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java [hunk 2]
```diff
@@ -102,12 +106,16 @@ public void setAveragePower(final double averagePower) {
         this.averagePower = averagePower;
     }
 
+    private void computeAveragePower(final Duration deltaTime) {
```
Comment (michelececcacci) [inline src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java:109]: can you please apply my previous comment here?

**Review by michelececcacci — CHANGES_REQUESTED** [review]
use a DurationConverter

File: src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java (hunk 2 above)
Comment (aleemont1) [reply src/main/java/homer/controller/impl/electricalmeter/ElectricalMeterImpl.java:109]: Yep I didn't read it, on it now.

**Comment by michelececcacci** [comment]
LGTM

**Pull request merged**